README.adoc

```diff
@@ -1,6 +1,6 @@
 = Install Alpine Linux in chroot
 :script-name: alpine-chroot-install
-:script-sha1: dcceb34aa63767579f533a7f2e733c4d662b0d1b
+:script-sha1: 4460336640193d0f1ce0a6e1f63ff903644712bd
```

Comment:
This should be updated only together with `version`, i.e. when a new release is made, and it’s done by `make release`.